src/utils/interpret-svg.js

```diff
@@ -303,13 +303,39 @@ var applySvgAttributes = function(node, elem, parentStyles) {
       case 'className':
         elem.classList = value.split(' ');
         break;
+      case 'x':
+      case 'y':
+        elem.translation[key] = parseFloat((value + '').replace('px', ''));
```

Comment:
`parseFloat` silently ignores non-numeric characters, so:
- Stripping `px` from the string is unnecessary
- All values will be interpreted with pixel units, so `2mm` will be treated as 2 pixels, `100%` will be 100 pixels, etc. which may not be desired

Reply:
Not desired, but this is the way it is for <rect /> tags already.